packages/admin-ui-docs/src/theme/DocPage/index.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Wut? Normal this?

Reply:
This component is a internal component of docusaurus, we are ejecting it to be able to customize it. That is the reason the comment block is shown